Gemfile

```diff
@@ -27,3 +27,6 @@ group :test do
   gem "poltergeist"
 end
 
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin]
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
looks great, i wonder why these two lines don't use the same platform directive though. seems like they should both use the same conditional..?